sigh/lib/sigh/resign.rb

```diff
@@ -8,7 +8,9 @@ def run(options, args)
 
       ipa, signing_identity, provisioning_profiles, entitlements, version, display_name, short_version, bundle_version, new_bundle_id = get_inputs(options, args)
       # ... then invoke our programmatic interface with these vars
-      resign(ipa, signing_identity, provisioning_profiles, entitlements, version, display_name, short_version, bundle_version, new_bundle_id)
+      success = resign(ipa, signing_identity, provisioning_profiles, entitlements, version, display_name, short_version, bundle_version, new_bundle_id)
+
+      raise "Resign failed" unless success
```

Comment:
Please use `UI.user_error!` instead of `raise` here. 

The `resign` action has a [good example](https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/resign.rb#L12) of an error message for this so let's be consistent and use
```ruby
UI.user_error!("Failed to re-sign .ipa")
````